### ISSUE TITLE
[Bug]: lr_sha for 2024

### ISSUE THREAD

**Issue by metallmanu** [mampfes/hacs_waste_collection_schedule#1505]
### I Have A Problem With:

A specific source

### What's Your Problem

Calender for 2024 not available in Homeassistant for lr_sha

### Source (if relevant)

lr_sha

### Logs

```Shell
no relevant logs
```

### Checklist Source Error

- [X] Use the example parameters for your source (often available in the documentation) (don't forget to restart Home Assistant after changing the configuration)
- [X] Checked that the website of your service provider is still working
- [X] Tested my attributes on the service provider website (if possible)
- [X] I have tested with the latest version of the integration (master) (for HACS in the 3 dot menu of the integration click on "Redownload" and choose master as version)

### Checklist Sensor Error

- [X] Checked in the Home Assistant Calendar tab if the event names match the types names (if types argument is used)

### Required

- [X] I have searched past (closed AND opened) issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by TMan1695** [comment]
It looks like the homepage lrsha.de has changed:
https://www.lrasha.de/de/buergerservice/abfallwirtschaft/abfallkalender

**Comment by TMan1695** [comment]
They have added new functions to the website between the last time I checked (5 days ago) and today. Now you are able to export ICS files, which currently work with Waste Collection schedule. You have to proceed as follows:

1. go to https://www.lrasha.de/de/buergerservice/abfallwirtschaft/abfallkalender
2. Choose whatever you are interested in (areas, type of trash, etc.)
3. go to "importieren" 
4. copy the link on the little "ics" button
5. go to your HA config and add it as a source under waste collection schedule, see code below
6. restart HA, you should be good to go.


```
waste_collection_schedule:
  sources:
    - name: ics
      args:
        url: https://api.cross-7.de/public/calendar/396/events/ics?CategoryIds=xxxxx&SecondCategoryIds=xxxxx
```

If I find some time over the holidays, I might be able to fix it in the "automatic" version of Waste Collection Schedule.

**Comment by metallmanu** [comment]
Wow, thank you! Worked for me

**Comment by metallmanu** [comment]
> Wow, thank you! Worked for me

**Comment by TMan1695** [comment]
Looks like it will be fixed with this commit:
https://github.com/mampfes/hacs_waste_collection_schedule/pull/1607
Can be closed afterwards!

**Comment by metallmanu** [comment]
> Looks like it will be fixed with this commit: #1607 Can be closed afterwards!
